core/Cargo.toml

```diff
@@ -20,6 +20,7 @@ circle-ci = { repository = "embedded-graphics/embedded-graphics", branch = "mast
 [dependencies]
 nalgebra = { version = "0.23.0", optional = true, default-features = false }
 byteorder = { version = "1.3.4", default-features = false }
+defmt = { version = "0.2.1", optional = true, git = "https://github.com/derekdreery/defmt", branch = "impl_format_for_range"}
```

Comment:
For consistency with e-g itself, IMO it would be a good idea to add a `defmt_support = [ "defmt" ]` feature to `core` as well.

Reply:
Could we just rename the future to `defmt`? https://rust-lang.github.io/api-guidelines/naming.html#c-feature

Reply:
I tried this, but a feature can't have the same name as a crate. I guess we could alias the crate to free up the feature name?

Reply:
An optional crate can be used as a feature without explicitly defining a feature in `Cargo.toml`.

Reply:
Sure, but the feature in the not-`core` `Cargo.toml` is there because it turns on `core`'s `defmt` feature too, so we can't just use the `defmt` optional dependency there. 

I don't think there's a solution to requiring the `defmt_support` feature in the root `Cargo.toml`, hence me suggesting we add the `defmt_support` feature here too.

Reply:
OK, I hadn't that the `defmt` feature in e-g would also need to enable the same feature in `core`. So forget what I said and just use a `defmt_support´ flag in both crates.

Reply:
Now we can have `defmt = [ "dep:defmt", "embedded-graphics-core/defmt" ]`